third_party/lint/repos.bzl

```diff
@@ -17,3 +18,9 @@ def lint_repos(lowrisc_lint = None):
         build_file = Label("//third_party/lint:BUILD.protolint.bazel"),
         url = "https://github.com/yoheimuta/protolint/releases/download/v0.50.5/protolint_0.50.5_linux_amd64.tar.gz",
     )
+    deb_package(
+        name = "clang-format",
+        url = "http://ftp.us.debian.org/debian/pool/main/l/llvm-toolchain-14/clang-format-14_14.0.6-12_amd64.deb",
+        sha256 = "3fbcadd614577667f6ff5958cc70b01fcf0b6a27f723a01516ced294b83d1002",
+        build_file = Label("//third_party/lint:BUILD.clang-format.bazel"),
+    )
```

Comment:
Nice.  My only concern is whether or not we have to worry about deps of the debian package.  Since our development environment is already ubuntu (a debian derivative), I suspect not.